Sources/UI/SwiftUI/Views/LottieView/LottieView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LottieView.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 02/03/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+#if canImport(Lottie)
+import Lottie
+import SwiftUI
+
+@available(iOS 15.0, *)
+public struct LottieView: UIViewRepresentable {
```

Comment:
Lottie supports SwiftUI officially since v4.3.0.
https://github.com/airbnb/lottie-ios/discussions/2189

Reply:
Yeah, I know. It is just cumbersome to import lib every time I need it. But makes sense yes. Redundant.